connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -102,6 +103,10 @@ private GeyserConnector(PlatformType platformType, GeyserBootstrap bootstrap) {
         Toolbox.init();
         Translators.start();
 
+        if (platformType != PlatformType.STANDALONE) {
```

Comment:
This check should happen in standalone, in case someone is running standalone in its own container.